docs/learning-powershell/README.md

```diff
@@ -35,7 +35,7 @@ At the end of this exercise, you should be able to launch the PowerShell session
 
 Getting Started with PowerShell
 ----
-PowerShell commands follow a Verb-Noun semantic with a set of parameters.
+PowerShell commands follow a verb-noun semantic with a set of parameters.
```

Comment:
This should remain capitalized since the PowerShell cmdlet naming convention is Capitalized Verb-Noun (i.e. `Get-Process` not `get-process`).